Tests/ClientRuntimeTests/NetworkingTests/URLSession/FoundationStreamBridgeTests.swift

```diff
@@ -52,15 +47,21 @@ class FoundationStreamBridgeTests: XCTestCase {
             // Run a test for every possible data size up to the maximum
             let dataSize = min(run, maxDataSize)
 
-            // The buffer may be as small as 1 byte, up to 2x as big as the data size capped by maxBufferSize
-            let bufferSize = Int.random(in: 1...min(2 * dataSize, maxBufferSize))
+            // Sizes of the following buffers within the `FoundationStreamBridge` under test are randomized to
+            // try and elicit errors in the streaming process.
+
+            // The bridge buffer may be as small as 1 byte, up to 2x as big as the data size
+            let bridgeBufferSize = Int.random(in: 1...(2 * dataSize))
+
+            // The bound stream buffer may be as small as 1 byte, up to 2x as big as the data size
+            let boundStreamBufferSize = Int.random(in: 1...(2 * dataSize))
 
             // Fill a data buffer with dataSize random numbers
             let originalData = Data(bytes: randomBuffer, count: dataSize)
 
             // Create a stream bridge with our original data & open it
             let bufferedStream = BufferedStream(data: originalData, isClosed: true)
-            let subject = FoundationStreamBridge(readableStream: bufferedStream, bufferSize: bufferSize, logger: TestLogger())
+            let subject = FoundationStreamBridge(readableStream: bufferedStream, bridgeBufferSize: bridgeBufferSize, boundStreamBufferSize: boundStreamBufferSize, logger: TestLogger())
```

Comment:
Create the `FoundationStreamBridge` under test with individual settings for bridge buffer size and bound stream buffer size.